<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvaluationTool</class>
 <widget class="QWidget" name="EvaluationTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evaluate Traces</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="listView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>707</width>
          <height>464</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Calculate Metrics</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_calculateMetrics">
               <property name="text">
                <string>Calculate metrics</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_genPlots">
               <property name="text">
                <string>Generate plots</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_exportCSV">
               <property name="text">
                <string>Export to CSV</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="TraceMetricTreeWidget" name="traceMetricTreeWidget">
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TraceMetricTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>presentation/tracemetrictreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
